src/components/contact/contact.container.js

```diff
@@ -0,0 +1,31 @@
+import { connect } from 'react-redux';
+import Contact from './contact.component';
+import {
+  changeIsChecked,
+  deleteContact,
+  increaseLikes,
+  decreaseLikes,
+  fetchFavourites
+} from '../../actions';
+
+const mapStateToProps = (state, ownProps) => ({
+  contact: state.contactsData.contacts.find(x => x.id === ownProps.id)
```

Comment:
This is not good! This parses the array for each contact. Pretty hard performance hit. In this case you can just leave it as it was before, `foreach(contact in contacts) -> render component`

Reply:
And send the `contact` with props from contacts-list